pkg/ingester/ingester_v2_test.go

```diff
@@ -743,3 +744,85 @@ func newIngesterMockWithTSDBStorage(ingesterCfg Config, registerer prometheus.Re
 
 	return ingester, cleanup, nil
 }
+
+func TestIngester_V2Startup(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name  string
+		seed  func(string)
```

Comment:
Maybe calling this `setup` could be more clear.

Reply:
Fixed